Problem4/src/problem4/Problem4.java

```diff
@@ -14,8 +15,61 @@
     /**
      * @param args the command line arguments
      */
-    public static void main(String[] args) {
-        // TODO code application logic here
-    }
-    
-}
+    public static void main(String[] args)
+    {
+        try
+        {
+            File inputFile = new File("myTextWindows.txt"); //Declare Variables
+            File outputFile = new File("myTextData.txt");
+            if (!outputFile.exists())
+            {
+                outputFile.createNewFile();
+            }
+            BufferedReader br = new BufferedReader(new FileReader("mytextWindows.txt"));
+            PrintWriter pw = new PrintWriter (new BufferedWriter(new FileWriter("mytextData.txt", false)));
```

Comment:
You create File objects for inputFile and outputFile, but don’t use them when creating the FileReader and FileWriter objects.